Services/CO.CDP.EntityVerification/Api/PponEndpointExtensions.cs

```diff
@@ -34,7 +34,31 @@ await useCase.Execute(new LookupIdentifierQuery(identifier))
                 operation.Responses["404"].Description = "Identifier not found.";
                 operation.Responses["500"].Description = "Internal server error.";
                 return operation;
-            });
+            });       
+    }
+
+    public static void UseRegistriesEndpoints(this WebApplication app)
+    {
+        app.MapGet("/registries/{countrycode}",
+           [Authorize(Policy = "OneLoginPolicy")]
+        async (string countrycode, IUseCase<string, IEnumerable<CountryIdentifiers>> useCase) =>
+           await useCase.Execute(countrycode)
+                  .AndThen(organisation => organisation != null ? Results.Ok(organisation) : Results.NotFound()))
```

Comment:
It isn't an organisation we're dealing with here.

Also, can the result of the use case actually be `null`? It's not reflected in the type.

Reply:
Done